### PR TITLE
AD review adjustments

### PR DESCRIPTION
**Pull request by dericed**
Here's more work towards the recent AD review.

### CODE REVIEW

File: specification.markdown [hunk 1]
```diff
@@ -130,7 +130,7 @@ Octet Length | Size | Representation
 4            | 2^28 | 0001 xxxx xxxx xxxx xxxx xxxx xxxx xxxx
 5            | 2^35 | 0000 1xxx xxxx xxxx xxxx xxxx xxxx xxxx xxxx xxxx
 
-Data encoded as a `Variable Size Integer` MAY be rendered at octet lengths larger than needed to store the data. In this table a binary value of `0b10` is shown encoded as different `Variable Size Integers` with lengths from one octet to four octet. All four encoded examples have identical semantic meaning though the `VINT_WIDTH` and the padding of the `VINT_DATA` vary.
+Data encoded as a `Variable Size Integer` are permitted to be rendered at octet lengths larger than needed to store the data. In this table a binary value of `0b10` is shown encoded as different `Variable Size Integers` with lengths from one octet to four octet. All four encoded examples have identical semantic meaning though the `VINT_WIDTH` and the padding of the `VINT_DATA` vary.
```
[the remark this answers is not in the document]
Comment (dericed) [reply specification.markdown]: Responds to this AD comment:

>> §5.4, paragraph after first table: "Data encoded as a "Variable Size Integer" MAY be rendered at octet lengths larger than needed to store the data."

> Is that intended as permission, or a statement of fact? If the latter, then the normative keyword is not appropriate.

Better suggestion?

[the remark this answers is not in the document]
Comment (mbunkus) [reply specification.markdown]: I consider this to be a permission with the default and desired behavior to render at minimum possible length. Implementations are permitted to chose longer lengths, e.g. in order to reserve space for being able to overwrite the element later.

This seems to be the perfect case for the normative keyword MAY — meaning leave it as is.

[the remark this answers is not in the document]
Comment (dericed) [reply specification.markdown]: I am okay to revert here, but could we adjust the wording in a way that address the doubt that the AD felt at reading this :`Is that intended as permission, or a statement of fact?`

[the remark this answers is not in the document]
Comment (mbunkus) [reply specification.markdown]: But doesn't the use of MAY imply that permission already? Wouldn't we have to adjust all uses of MAY then?

One possible wording, if we have to make it clearer that this is indeed intentional and a permission, might be:

> Data encoded as a `Variable Size Integer` MAY be rendered at octet lengths larger than needed to store the data in order to facilitate overwriting it at a later date, e.g. when its final size isn't known in advance.

[the remark this answers is not in the document]
Comment (dericed) [reply specification.markdown]: updated per @mbunkus' suggestion

**Comment by dericed** [comment]
will merge soon, thx for the reviews

**Pull request merged**